modules/apps/web-experience/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_sharing.jsp

```diff
@@ -79,6 +73,12 @@ String widgetURL = PortalUtil.getWidgetURL(portlet, themeDisplay);
 							<textarea class="field form-control lfr-textarea" id="<portlet:namespace />widgetScript" onClick="this.select();" readonly="true"><%= HtmlUtil.escape(textAreaContent) %></textarea>
 						</aui:field-wrapper>
 
+						<c:if test='<%= Validator.isNotNull(xFrameOptionsHeader) && (xFrameOptionsHeader.indexOf("ALLOW-FROM") == -1) %>'>
```

Comment:
Whoops, I thought I changed that to ```!=```.  Might have forgotten to commit that change.  I'll update it right now.